### DISCUSSION TITLE
Welcome to fuzzy-c-means Discussions!

### DISCUSSION THREAD

**Discussion by omadson** [omadson/fuzzy-c-means#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Xiaozhaoyaonuli** [comment]
I am a graduate student in the field of hydrology. I am currently using fuzzy C-means to perform watershed zoning, but I have no idea where to start.
